packages/palette/src/elements/Carousel/CarouselStructure.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from "styled-components"
+import { system } from "styled-system"
+import { Box, BoxProps } from "../Box"
+
+const RAIL_TRANSITION_MS = 500
+
+const transition = system({ transition: true })
+
+/** CarouselRailProps */
+export type CarouselRailProps = BoxProps & { transition?: string }
+
+/** A `CarouselRail` slides back and forth within the viewport */
+export const CarouselRail = styled(Box)<CarouselRailProps>`
+  height: 100%;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  white-space: nowrap;
+  ${transition}
+`
+
+CarouselRail.defaultProps = {
+  display: "flex",
+  transition: `transform ${RAIL_TRANSITION_MS}ms`,
+}
+
+/** CarouselCellProps */
+export type CarouselCellProps = BoxProps
+
+/** A `CarouselCell` wraps a single child in the carousel */
+export const CarouselCell = styled(Box)``
```

Comment:
(non-blocking/thought) any reason this can't live in `Carousel`? Or do you think that things are different enough (and / or will expand enough) to warrant a new file?